scripts/us_census/pep/Population_Estimate_by_ASR/county_2000_2010.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This Python Script is
+for County Level Data
+2000-2010.
+'''
+
+import os
+import pandas as pd
+import numpy as np
+
+
+def county2000():
+    """
+    This Python Script Loads csv datasets
+    from 2000-2010 on a County Level,
+    cleans it and create a cleaned csv.
+    """
+    # Used to collect data after every loop for every file's df.
+    final_df = pd.DataFrame()
+    for i in range(1, 57):
+        if i not in [3, 7, 14, 43, 52]:
+            j = f'{i:02}'
+            url = 'https://www2.census.gov/programs-surveys/popest/datasets/2'+\
+                '000-2010/intercensal/county/co-est00int-alldata-'+str(j)+'.csv'
+            df = pd.read_csv(url, encoding='ISO-8859-1')
+            # Filter years 1 - 12.
+            df['Year'] = df['YEAR']
+            df.drop(columns=['YEAR'], inplace=True)
+            df = df.query("Year not in [1,12,13]")
+            # Filter by agegrp = 99.
+            df = df.query("AGEGRP != 99")
+            df['Year'] = df['Year'].astype(str)
+            # Replacing the numbers with more understandable metadata headings.
+            _dict = {
```

Comment:
Another instance of _dict being used in just one place?

Reply:
All the _dict were initially created for better readability. However I have moved them inside the replacement function itself.